instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesExtractorBuilder.java

```diff
@@ -18,25 +18,21 @@ public final class SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> {
   private static final AttributeKey<String> DB_SQL_TABLE = AttributeKey.stringKey("db.sql.table");
 
   final SqlClientAttributesGetter<REQUEST> getter;
-  AttributeKey<String> dbTableAttribute = DB_SQL_TABLE;
+  AttributeKey<String> oldSemconvTableAttribute = DB_SQL_TABLE;
   boolean statementSanitizationEnabled = true;
 
   SqlClientAttributesExtractorBuilder(SqlClientAttributesGetter<REQUEST> getter) {
     this.getter = getter;
   }
 
   /**
-   * Configures the extractor to set the table value extracted by the {@link
-   * SqlClientAttributesExtractor} under the {@code dbTableAttribute} key. By default, the <code>
-   * db.sql.table</code> attribute is used.
-   *
-   * @param dbTableAttribute The {@link AttributeKey} under which the table extracted by the {@link
-   *     SqlClientAttributesExtractor} will be stored.
+   * @deprecated not needed anymore since the new semantic conventions always use db.collection.name
    */
   @CanIgnoreReturnValue
-  public SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> setTableAttribute(
-      AttributeKey<String> dbTableAttribute) {
-    this.dbTableAttribute = requireNonNull(dbTableAttribute);
+  @Deprecated
+  public SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> setOldSemconvTableAttribute(
+      AttributeKey<String> oldSemconvTableAttribute) {
+    this.oldSemconvTableAttribute = requireNonNull(oldSemconvTableAttribute);
```

Comment:
Any reason this method is renamed? To preserve backwards compatibility shouldn't we keep the method with the original name and javadoc and just add the deprecation notice?